### DISCUSSION TITLE
网络错误？

### DISCUSSION THREAD

**Discussion by yangxiang556** [LSPosed/MagiskOnWSALocal#469] [Q&A]
COMMAND_LINE=--arch x64 --release-type retail --magisk-ver stable --gapps-brand MindTheGapps --gapps-variant pico  --root-sol magisk --compress    --compress-format 7z
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
INFO: Compress Format: 7z
Build: RELEASE_TYPE=Retail
Generate Download Links
Generating WSA download link: arch=x64 release_type=Retail
Traceback (most recent call last):
  File "/home/yangxiang/MagiskOnWSALocal/scripts/generateWSALinks.py", line 109, in <module>
    tmpdownlist = open(download_dir/tempScript, 'a')
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '../download/download.list'
Build: an error has occurred, exit

Cleanup Work Directory
Cleanup Download Directory

**Comment by CarlGao4** [comment]
Please run the script as root (use `sudo ./scripts/run.sh` instead of `./scripts/run.sh`)
